src/librustc_borrowck/borrowck/mod.rs

```diff
@@ -971,13 +978,11 @@ impl<'a, 'tcx> BorrowckCtxt<'a, 'tcx> {
                             let span = self.tcx.map.span(local_id);
                             if let Ok(snippet) = self.tcx.sess.codemap().span_to_snippet(span) {
                                 if snippet != "self" {
-                                    db.span_suggestion(
-                                        span,
-                                        &format!("to make the {} mutable, use `mut` as shown:",
-                                                 self.cmt_to_string(&err.cmt)),
-                                        format!("mut {}", snippet));
+                                    db.span_label(span,
+                                        &format!("use `mut {}` here to make mutable", snippet));
```

Comment:
We should chat about lint implications.  For the actual error, moving it to a label text appears to improve readability "at a glance" of the errors.  That said, we may want to distinguish them in some way that lints and autocorrection type tools can find and use them.